packages/v4/src/content/community/extensions-data.js

```diff
@@ -0,0 +1,58 @@
+export const communityExtensions = [
+  {
+    "component": "Catalog View",
+    "description": "Implements a catalog view using PatternFly components",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/catalog-view-item-header"
+      },
+      {
+        "name": "Code",
+        "href": "https://www.npmjs.com/package/@patternfly/react-catalog-view-extension"
+      }
+    ]
+  },
+  {
+    "component": "Console",
+    "description": "React-console extension",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/access-consoles"
+      },
+      {
+        "name": "Code",
+        "href": "https://github.com/patternfly/patternfly-react/tree/master/packages/react-console"
+      }
+    ]
+  },
+  {
+    "component": "Topology View",
+    "description": "Implements a topology view",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Code",
+        "href": "https://github.com/patternfly/patternfly-react/tree/master/packages/react-topology"
+      }
+    ]
+  },
+  {
+    "component": "Virtualized table",
+    "description": "A virtualized table that supports infinite scrolling",
+    "tags": ["App services"],
+    "links": [
+      {
+        "name": "Doc",
+        "href": "https://patternfly-react.surge.sh/extensions/virtual-scroll-table",
```

Comment:
Small copy edits for all topology view extension examples: 

- Rewrite last table column in sentence case across examples on "Virtual scroll table" and "Virtual scroll window scroller" pages: _Last commit_.
- Rewrite every example's description in sentence case: _Basic table, Sortable virtualized table, Selectable virtualized table, Actions virtualized table_.